compose/docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       - HARMONY_SETTING_MODEL_REGISTRY_ROOT=/model_registry
       - GROUP=default
       - PARTITION_KEY=default-partition
-      - CONTROL_PLANE_URL=http://control-plane:9000
+      - CONTROL_PLANE_URL=http://control-plane:9009
```

Comment:
in the control-plane service, let's set the env variable `ADAPTIVE_SERVER__BIND_INTERNAL` so that we easily know what this 9009 stands for.